tests/PHPStan/Rules/Arrays/data/appended-array-item.php

```diff
@@ -11,16 +11,28 @@ class Foo
 	/** @var callable[] */
 	private $callables;
 
-	public function doFoo()
+	public function accepted()
 	{
 		$this->integers[] = 4;
 		$this->integers['foo'] = 5;
-		$this->integers[] = 'foo';
-		$this->callables[] = [$this, 'doFoo'];
-		$this->callables[] = [1, 2, 3];
 		$this->callables[] = ['Closure', 'bind'];
 		$this->callables[] = 'strpos';
+		$this->callables[] = [$this, 'accepted'];
+	}
+
+	public function notAccepted1()
+	{
+		$this->integers[] = 'foo';
+		$this->callables[] = [1, 2, 3];
+	}
+
+	public function notAccepted2()
```

Comment:
if you append new tests at the end of the file, pre-existing test-expectation line numbers don't need to be changed
(instead of adding them in between)

Reply:
there are no new tests, just moved existing ones around :/